lib/wallet/payment_history_list_item.dart

```diff
@@ -121,6 +131,25 @@ class PaymentHistoryListItem extends StatelessWidget {
     );
   }
 
+  Transform receiveOffChainIcon() {
+    return Transform.rotate(
+        angle: -135 * math.pi / 180,
+        child: Icon(
+          FontAwesomeIcons.arrowLeft,
+          color: Colors.green[700],
+        ));
+  }
+
+  Transform sendOffChainIcon() {
+    return Transform.rotate(
+      angle: 135 * math.pi / 180,
+      child: Icon(
+        FontAwesomeIcons.arrowLeft,
+        color: Colors.red[700],
```

Comment:
I don't think another shade of `green` / `red` will look good, I think we have enough colors. I'd rather use a different icon, or just use the same color and icon and distinguish by subtitle only.

Reply:
This is pretty much the same shade, just slightly different tone. I will just use the same colour for now, for the sake of this getting in. Feel free to change to whatever you want later on.